collections/_article/january-jams-2024.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "January Jams"
+excerpt: "Looking back at a month of GameJams, we curated a list of ten events that you might or might not have heard of before."
```

Comment:
```suggestion
excerpt: "Looking back at a month of game jams, we curated a list of ten events that you might or might not have heard of before."
```
Again, a matter of subjective taste, but "GameJams" made it sound like that was a particular branded thing.

Reply:
Same here, it looks better the way you suggested it. Thank you!